### ISSUE TITLE
There are only 3 buttons - the "STOCKS" button is missing

### ISSUE THREAD

**Issue by andrewkhomski** [andrewkhomski/CPLPLUS#42]
There are only 3 buttons - the "STOCKS" button is missing.

See attached...

![issues-23-12282015](https://cloud.githubusercontent.com/assets/15894297/12029131/b078f00e-adb1-11e5-89cc-05ba3d3822d7.png)

**Comment by davidm-finmc** [comment]
Yeah that’s intentional, but you can’t tell from the getoffer.php file – the buttons are in a separate html file.

**Comment by davidm-finmc** [comment]
If the button on top were left-aligned, so there was an empty top-right-corner, then I would want you to report a bug. But if it is centered (like it is now), then it has to be by design (intentional).
